cmd/neofs-node/closer.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+type closer struct {
+	name string
+	fn   func()
+}
+
+func getCloser(c *cfg, name string) *closer {
+	for _, closer := range c.closers {
+		if closer.name == name {
+			return closer
+		}
+	}
+	return nil
+}
+
+func delCloser(c *cfg, name string) {
+	for i, clsr := range c.closers {
+		if clsr.name == name {
+			c.closers[i] = c.closers[len(c.closers)-1]
+			c.closers[len(c.closers)-1] = nil
```

Comment:
Do we really need this according to next line?

Reply:
Not sure how it should be in golang, the idea is to facilitate gc work.